apps/src/music/views/timeline.module.scss

```diff
@@ -15,6 +14,7 @@
     overflow-y: hidden;
     height: 100%;
     position: relative;
+    background-color: $music_black;
```

Comment:
This one's interesting: without it, Chrome on macOS picked a bad color for its scrollbars, even though it was inheriting the same background color.